pkg/accounting/accounting.go

```diff
@@ -439,7 +439,7 @@ func (c *creditAction) Cleanup() {
 	defer c.accountingPeer.lock.Unlock()
 
 	if c.price.Cmp(c.accountingPeer.reservedBalance) > 0 {
-		c.accounting.logger.Error(nil, "attempting to release more balance than was reserved for peer")
+		c.accounting.logger.Error(ErrOverRelease, ErrOverRelease.Error(), "peer", c.peer)
```

Comment:
same here, I would replace ErrOverRelease.Error() with some meaningful msg

Reply:
it should be nil as Peter said because the error was not returned to us, so we can simply extract the string from error using .Error()